src/logging/log.ts

```diff
@@ -0,0 +1,228 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/**
+ * Use of `any` in this module is generally deliberate to help with generics
+ */
+import bunyan = require('bunyan');
```

Comment:
It may be smarter to just expose an interface that can be served by bunyan, rather than adding bunyan as a direct dependency. i.e. port over LogContext etc... but the actual implementation for the logging can still be specific to the environment (i.e. `hpc_service`, `hpc-api`) etc... Which will give a bit more flexibility in terms of dependencies on these libraries, where `hpc-api` will likely be a bit more flexible.